docs/installation/installing-with-npm.md

```diff
@@ -15,7 +15,14 @@ To use GOV.UK Frontend with NPM you must:
    you don’t already have one. You can create a default `package.json` file by
    running `npm init` from the root of your application.
 
-3. If you want to use the GOV.UK Frontend Nunjucks macros, install Nunjucks -
+3. Install either:
+
+   - [LibSass](https://sass-lang.com/libsass) - version 3.3.0 or higher
```

Comment:
Do we think it's worth documenting in some way the issue that GOV.UK have seen with LibSass and Rails? https://github.com/alphagov/finder-frontend/issues/1124

Reply:
Seems like it's a common issue people will run into :+1:

Reply:
Happy for that to be done later if it's going to block this otherwise.

Reply:
Happy to add something! Would this work?

"If your app uses Rails, we recommend not using SassC::Rails because it may cause errors. Install Dart Sass instead." 

I appreciate there are other options, but installing Dart Sass seems like the simplest route?

Reply:
Maybe we should raise an issue and do it later just cos it's a bit nuanced...

Reply:
Sounds good - let's go with what we have and look at the LibSass and Rails issue later.

Reply:
In which case I think this is good to 🚀